tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -0,0 +1,70 @@
+  <PropertyGroup>
+    <_CoreCompileOverrideDependsOn Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">Pack</_CoreCompileOverrideDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CoreCompile" DependsOnTargets="$(_CoreCompileOverrideDependsOn)" />
+
+  <Target Name="AddSingleConfigBuildOutput"  Returns="@(_PackageContentsForPlat)">
+    <MSBuild Projects="$(SolutionPath)"
+       Targets="GetSolutionConfigurationContents"
+       Properties="Configuration=$(Configuration);Platform=$(Platform)">
+      <Output TaskParameter="TargetOutputs" ItemName="_SolutionConfigurationContents" />
+    </MSBuild>
+
+    <Message Text="AddSingleConfigBuildOutput Platform: $(Platform)" Importance="High"/>
+
+    <MSBuild Projects="$(MSBuildProjectFile)"
+       Targets="GetPackageContents"
+       Properties="SkipGetPackageContentsForOtherPlatforms=true;BuildingInsideVisualStudio=false;BuildProjectReferences=true;IsPackable=false;Configuration=$(Configuration);Platform=$(Platform);CurrentSolutionConfigurationContents=%(_SolutionConfigurationContents.Identity);GetPackageContentsDependsOn=Build;$(GetPackageContentsDependsOn)">
+      <Output TaskParameter="TargetOutputs" ItemName="_PackageContentsForPlat" />
+    </MSBuild>
+  </Target>
+
+  <Target Name="AddOtherConfigsBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' AND '$(Platform)' == 'AnyCPU'">
+    <ItemGroup>
+      <_PackTargetPlat Include="%(ProjectConfiguration.Platform)" Condition="'%(ProjectConfiguration.Configuration)' == '$(Configuration)' AND '%(ProjectConfiguration.Platform)' != '$(Platform)'" />
+    </ItemGroup>
+
+    <MSBuild Projects="$(MSBuildProjectFile)"
+       Targets="AddSingleConfigBuildOutput"
+       Properties="Platform=%(_PackTargetPlat.Identity)">
+      <Output TaskParameter="TargetOutputs" ItemName="_OtherPackageContents" />
+    </MSBuild>
+
+    <ItemGroup>
+      <PackageFile Include="@(_OtherPackageContents)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' and '$(PresetPackageContents)' == 'true'">
+    <ItemGroup>
+      <PackageFile Include="$(MSBuildProjectDirectory)\$(PackageId).props" Condition="Exists('$(MSBuildProjectDirectory)\$(PackageId).props')">
+        <PackagePath>build\$(PackageId).props</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildProjectDirectory)\$(PackageId).targets" Condition="Exists('$(MSBuildProjectDirectory)\$(PackageId).targets')">
+        <PackagePath>build\$(PackageId).targets</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildThisFileDirectory)Package.Linker.targets" Condition="Exists('$(MSBuildThisFileDirectory)Package.Linker.targets')">
+        <PackagePath>build\$(PackageId).Linker.targets</PackagePath>
+      </PackageFile>
```

Comment:
Not such what you're pointing at here

---
In reply to: [100939995](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100939995) [](ancestors = 100939995)